src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,8 +11,8 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_AGE = new Prefix("ag/");
```

Comment:
Is it a good idea to use two letters for the prefix for age? Maybe can consider using other letters like "y/" to denote years.